kolibri/plugins/management/assets/src/app.js

```diff
@@ -47,7 +48,7 @@ class ManagementModule extends KolibriModule {
     });
 
     router.start(rootvue, 'rootvue');
-    kolibri.coreActions.currentLoggedInUser(store, kolibri);
+    coreActions.currentLoggedInUser(store, kolibri);
```

Comment:
The main advantage is that it lets us maintain resolutions during testing, without having to have the core Kolibri object around during tests. If we just let the aliases resolve to the proper files, then everything that is needed to run the unit tests will be requireable, without having to mock up a core Kolibri object every time.